.github/workflows/generate_sample.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate sample project
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  test:
+    name: Generate sample project
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+
+      - name: Remove sample project
+        run: |
+          rm -rf sample
+
+      - name: Generate new sample project
+        run: |
+          dart pub global activate mason_cli
+          mason get
+          mason make template -c mason-config.json
+          # Move the generated project to the root directory for next steps & cleanup to not affect static code analysis
+          rsync -av --remove-source-files flutter_templates/ ./sample
+          rm -rf bricks/template
```

Comment:
@luongvo why don't we update the project name in `mason-config.json` to `sample`? Then we don't need the cleaning commands.
```suggestion
          mason make template -c mason-config.json
```

Reply:
Mason generates a new project called `sample` in the root directory 😄 
<img width="1273" alt="image" src="https://github.com/nimblehq/flutter-templates/assets/60863885/1d5f4c9c-97af-4dcc-9a1a-d15dc30d4daa">

Reply:
@manh-t I don't want to change the `project_name` in the config publicly as I want to keep it as default options. Hence, I added a step to change the field in CD workflow instead https://github.com/nimblehq/flutter-templates/pull/196/commits/cc50c5b5993e8f9c560cd077b0f3f120a555fd09